Source/WebKit/WebProcess/WebProcess.h

```diff
@@ -533,6 +533,7 @@ class WebProcess : public AuxiliaryProcess
     friend class WebConnectionToUIProcess;
     void didReceiveMessage(IPC::Connection&, IPC::Decoder&) override;
     bool didReceiveSyncMessage(IPC::Connection&, IPC::Decoder&, UniqueRef<IPC::Encoder>&) override;
+    void didClose(IPC::Connection&) override;
```

Comment:
final?

Reply:
So now `WebProcess::didClose()` is no longer called. Leaving WebProcess running forever, in glib ports at least.

Reply:
Oh I saw the fix is landed in [257721@main](https://commits.webkit.org/257721@main); thanks for fixing!